packages/protocol/contracts/identity/FederatedAttestations.sol

```diff
@@ -0,0 +1,160 @@
+pragma solidity ^0.5.13;
```

Comment:
We should ask Martin whether he recommends using a newer version of Solidity. Among other things, solidity ^0.8.0 has built in SafeMath. However, I'm not sure if updating just one contract to the newer version would break our tooling